scripts/nope.py

```diff
@@ -332,6 +333,25 @@ def main():
     else:
         logging.basicConfig(level=logging.INFO)
 
+    # if running in upload mode - immediately load JSON, upload, and exit
+    UPLOAD_FILE = args.upload_file
+    if UPLOAD_FILE != '':
```

Comment:
```suggestion
    if UPLOAD_FILE is not None:
```

I am getting error as:
```
$ ./nope.py
Traceback (most recent call last):
  File "/Users/memodi/workspaces/repos/ocp-qe-perfscale-ci/scripts/./nope.py", line 339, in <module>
    upload_file_path = DATA_DIR + '/' + UPLOAD_FILE
TypeError: can only concatenate str (not "NoneType") to str
```

it would be also better to print `help` showing all arguments if no arguments are passed.